### ISSUE TITLE
File modification times

### ISSUE THREAD

**Issue by zenhack** [zombiezen/filedrop#17]
It would be nice if FileDrop displayed modification times for files. Use case: I currently have a filedrop [grain][1] that contains the latest spk for [sandcal][2], but @ocdtrekkie mentioned the other day that it's hard to tell whether it's been updated with the latest changes.

[1]: https://thelibrary.sandcats.io/shared/evMWazczccSr8-8y4eG8Hc62wBsnukQX2XItoR_rd-k
[2]: https://github.com/zenhack/sandcal

**Comment by ocdtrekkie** [comment]
Indeed! I was torn between assuming it was or wasn't updated, hesitant to nag Ian about it, and didn't want to go through all the trouble of downloading and uploading the SPK just to discover it wasn't updated. Seeing the file modification time would be super helpful!

**Comment by zombiezen** [comment]
This sounds like a good feature! Unfortunately, I don't have bandwidth to work on this project anymore, but I'd be happy to review a PR and push it up to the Sandstorm marketplace.
